File: Python/Beginners/calculo_simples.py
# Caio Cesar
# BEE 1010

# Precisa do split() pois o input irá receber 3 valores numa mesma linha, separados pelo espaço
lista1 = input().split()
lista2 = input().split()

# Conversão do tipo de dado (string para int e float) e seu devido cálculo que a questão pediu
valor1 = int(lista1[1])*float(lista1[2])
valor2 = int(lista2[1])*float(lista2[2])

soma = valor1 + valor2

print(f'VALOR A PAGAR: R$ {soma:.2f}')

File: C++/Beginner/calculo_simples.cpp
// Caio Cesar 16/11/23
// BEE 1010

#include <cstdio>

int main() {

    int codigo, quantidade;
    float valor_unitario, valor_total = 0;

    scanf("%d %d %f", &codigo, &quantidade, &valor_unitario);
    valor_total += (quantidade * valor_unitario);

    codigo = 0, quantidade = 0, valor_unitario = 0;

    scanf("%d %d %f", &codigo, &quantidade, &valor_unitario);

    valor_total += (quantidade * valor_unitario);

    printf("VALOR A PAGAR: R$ %.2f\n", valor_total);

    return 0;
}